### DISCUSSION TITLE
How To Play Hellfire In DevillutionX On Android Version?  (And How To Put Mods)

### DISCUSSION THREAD

**Discussion by ZarkTye** [diasurgical/DevilutionX#3344] [Q&A] [answered]

**Comment by qndel** [comment]
Was reading the readme too hard? ...
Or following the message displayed on screen? 🙄

**Comment by ZarkTye** [comment]
Sorry Now I Remember Lol 
But Have To Put Mods? on Android version

**Reply by qndel** [reply]
what mods ...

**Comment by ZarkTye** [comment]
(Mods For Diablo 1)

**Reply by qndel** [reply]
You can't.

**Reply by AJenbo** [reply]
You have to transfer the Hellfire mpq files to you device. See the install instructions here.
https://github.com/diasurgical/devilutionX/blob/master/docs/installing.md#installing

**Reply by ZarkTye** [reply]
Ok

**Comment by ZarkTye** [comment]
![Screenshot_20211031-160724](https://user-images.githubusercontent.com/81594191/139597875-3267f635-677a-45f0-9ffb-2a93190d9b7a.png)


Now Its Done

**Reply by qndel** [reply]
That's not a mod ... that's expansion 🙄

**Comment by ZarkTye** [comment, marked as answer]
I already know it can't be put on 
Because you said
anders jembo sent for me a link on how to put hell fire on diablo 1
